doc/src/manual/interfaces.md

```diff
@@ -5,6 +5,81 @@ A lot of the power and extensibility in Julia comes from a collection of informa
 receive those functionalities, but they are also able to be used in other methods that are written
 to generically build upon those behaviors.
 
+## [Instance Properties](@id man-instance-properties)
+
+### [Overloading `object.field`](@id overloading-object-field)
```

Comment:
Why are there two section headers here, rather than just one?  What else could go in the "Instance Properties" section?

Reply:
Nothing else. It just that "Instance Properties - Overloading `object.field`" feels a very long title compared to others in this section. Should I change it to just that one long title?